packages/spor-react/src/modal/FullScreenDrawer.tsx

```diff
@@ -18,6 +18,10 @@ import { Drawer } from "./Drawer";
 
 type DrawerPlacement = "top" | "right" | "bottom" | "left";
 
+type DrawerBodyProps = {
+  id?: string;
```

Comment:
Er det noen grunn til at denne ikke legges direkte til FullScreenDrawerProps?

Reply:
Hvis jeg i stedet setter for eks `bodyId` som props i `FullScreenDrawe`r og da setter så direkte i `DrawerBody` at `id={bodyId}` så vil id-en til `DrawerBody` alltid bli overskrevet og den vil ikke lenger få default id-en som blir satt av Chakra. Jeg er åpen for andre forslag, ville bare ikke fjerne muligheten for bruk av default-id

Reply:
Helt kurant det, behold det sånn!